### PR TITLE
README update

### PR DESCRIPTION
**Pull request by cafemoloko**
Names & links update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,120 +1,152 @@
 # DepthAI Open Source Hardware
-This repository contains Luxonis open sourced baseboards, including the DepthAI OpenCV AI Kits, and contains Altium design files, documentation, and pictures. This repository is for anyone interested in developing with the DepthAI SoM or anyone interested in understaning more about the embedded hardware that powers DepthAI. An open source, community KiCad folder is also available for sharing KiCad based designs and components. 
+This repository contains Luxonis open sourced baseboards, contains Altium design files, documentation, and pictures. This repository is for anyone interested in developing with the OAK SoM or anyone interested in understaning more about the embedded hardware that powers DepthAI. An open source, community KiCad folder is also available for sharing KiCad based designs and components.
```
Comment (Erol444) [inline README.md:2]:
```suggestion
This repository contains Luxonis open sourced baseboards, Altium design files, documentation, and pictures. This repository is for anyone interested in developing with the OAK SoM or anyone interested in understaning more about the embedded hardware that powers DepthAI. An open source, community KiCad folder is also available for sharing KiCad based designs and components. 
```

**Review by Erol444 — APPROVED** [review]
besides the typo, I agree we should only have information in one place - docs - so we don't need to keep 2 sources of info up-to-date. Thanks!

**Review by Luxonis-David — APPROVED** [review]

**Comment by Erol444** [comment]
@cafemoloko should we close this:)?

**Comment by cafemoloko** [comment]
@Erol444 yeah, looks like a forgotten draft :)

**Pull request closed without merging**